platform/vcs-impl/src/com/intellij/openapi/vcs/changes/ControlledCycle.java

```diff
@@ -22,7 +22,7 @@
   private final AtomicBoolean myActive;
 
   public ControlledCycle(@NotNull Project project,
-                         final Supplier<Boolean> callback,
+                         final BooleanSupplier callback,
```

Comment:
There is only one place in the code where the instance of this public class is created, and it uses a lambda to express the `callback` parameter, so this particular change shouldn't break anything.